src/main/java/me/botsko/prism/database/mysql/MySQLPrismDataSource.java

```diff
@@ -54,10 +75,39 @@ public void setFile() {
 
     @Override
     public SelectQuery createSelectQuery() {
-        if (nonStandardSQL) {
+        if (nonStandardSql) {
             return new MySQLSelectQueryBuilder(this);
         } else {
             return new SQLSelectQueryBuilder(this);
         }
     }
+
+    private void detectNonStandardSql() {
+        try (
+                PreparedStatement st = getConnection().prepareStatement("SHOW VARIABLES");
+                ResultSet rs = st.executeQuery()) {
+            while (rs.next()) {
+                dbInfo.put(rs.getString(1).toLowerCase(), rs.getString(2));
+            }
+        } catch (SQLException e) {
+            Prism.debug(e.getMessage());
+        }
+        String version = dbInfo.get("version");
+        String versionComment = dbInfo.get("version_comment");
+
+        Prism.log("Prism detected you database is " + version + " / " + versionComment);
+        if (version.toLowerCase().contains("maria")
```

Comment:
Personally, I don't like the idea of just blacklisting MariaDB, it should instead just test if `SELECT ANY_VALUE(1)` works and set it `NonStandardSql: true` if it succeeds. This has already been reported to MariaDB (https://jira.mariadb.org/browse/MDEV-12481) and may someday become a valid function.